lib/Solver/STPSolver.cpp

```diff
@@ -293,9 +293,11 @@ runAndGetCexForked(::VC vc, STPBuilder *builder, ::VCExpr q,
   int pid = fork();
   // - error
   if (pid == -1) {
-    klee_warning("fork failed (for STP) - %s", llvm::sys::StrError(errno).c_str());
-    if (!IgnoreSolverFailures)
-      exit(1);
+    std::string msg = "fork() failed for STP - " + llvm::sys::StrError(errno);
+    llvm::errs() << msg << "\n";
+    if (IgnoreSTPFailures)
+      klee_warning("%s", msg.c_str());
+    else klee_error("%s", msg.c_str());
```

Comment:
Why are you printing the error with both `llvm::errs()` and `klee_warning/klee_error`?